src/test/java/Practice.java

```diff
@@ -0,0 +1,63 @@
+import Pages.AllSongsPage;
+import Pages.LoginPage;
+import Pages.ProfilePage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.Random;
+
+public class Practice extends BaseTest {
+    @Test
+
+    public void changeTheme(){
+        LoginPage loginPage = new LoginPage(getThreadLocal());
+        ProfilePage profilePage = new ProfilePage(getThreadLocal());
+
+        loginPage.login("stella_26021987@mail.ru", "te$t$tudent");
+        profilePage.clickAvatar();
+        profilePage.isProfilePageOpen();
+        profilePage.clickRandomTheme();
+        Assert.assertEquals(profilePage.clickRandomTheme(), profilePage.assertThemeChanged());
+    }
+    @Test
+
+    public void likeSong() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.